src/Plugins/Pagination.js

```diff
@@ -99,6 +99,28 @@ class Pagination {
 	}
 
 	isFiltered(value) {
+		const hasInclude = "include" in this.data.pagination;
+		const hasExclude = "exclude" in this.data.pagination;
+		if (hasInclude && hasExclude) {
+			throw new Error("Pagination cannot have both `include` and `exclude` filters.");
+		}
+		if (hasInclude) {
+			let included = this.data.pagination.include;
+			if (Array.isArray(included)) {
+				return included.indexOf(value) === -1;
```

Comment:
I think using `.includes()` makes more sense in this scenario.
```suggestion
				return !included.includes(value);
```